.travis/deploy.sh

```diff
@@ -10,6 +10,7 @@ function message() {
     echo
 }
 
+ENVIRONMENT=$1
```

Comment:
Sure does need to be moved up. How did this happen?

Reply:
> Sure does need to be moved up. How did this happen?

You are correct, it does need to move up. It gets passed in through the travis.yml file when the script is called.